sql/telemetry_derived/nondesktop_day_2_7_activation/query.sql

```diff
@@ -0,0 +1,61 @@
+WITH base AS (
```

Comment:
Per https://github.com/mozilla/bigquery-etl#recommended-practices this table name should have a `_v1` suffix (`telemetry_derive/nondesktop_day_2_7_activation_v1/query.sql `) and we should also provide a user-facing view in `telemetry/nondesktop_day_2_7_activation/view.sql`.

Reply:
And to be consistent with existing table naming, this should probably be prefixed with `firefox_nondesktop` rather than must `nondesktop`.

Reply:
Above changes have been made